src/client.py

```diff
@@ -30,12 +34,41 @@ def connect():
     """Get player name from user and request to server to join game."""
     message = "Enter name: "
     name = prompt_user(message)
-    response = requests.post(GAME_URL, json={"name": name})
-    if response.status_code == requests.codes.created:
-        game_id = response.json()["game_id"]
-        return name, game_id
+    message = (
```

Comment:
Maybe instead prompt:

    Select number of players:
    1 player  => join existing game
    2 players => create new two player game
    3 players => create new three player game

This would help give the variable a more meaningful name.